src/libData/AccountData/AccountStoreSC.tpp

```diff
@@ -118,28 +122,197 @@ void AccountStoreSC<MAP>::InvokeInterpreter(
+          LOG_GENERAL(
+              WARNING,
+              "Exception thrown trying to update Contract code " << e.what());
+        }
+        // Actually Update the state for the contract
+        try {
+          for (const auto& sit : it->Storage()) {
+            if (!Contract::ContractStorage::GetContractStorage()
+                     .UpdateStateValue(
+                         Address(it->Address()),
+                         DataConversion::StringToCharArray(sit.Key()), 0,
+                         DataConversion::StringToCharArray(sit.Value()), 0)) {
+              return false;
+            }
+          }
+        } catch (std::exception& e) {
+          // for now catch any generic exceptions and report them
+          // will exmine exact possibilities and catch specific exceptions.
+          LOG_GENERAL(WARNING,
+                      "Exception thrown trying to update state on the contract "
+                          << e.what());
+        }
+        try {
+          if (it->Balance().size())
+            targetAccount->SetBalance(uint128_t(it->Balance()));
+        } catch (std::exception& e) {
+          // for now catch any generic exceptions and report them
+          // will exmine exact possibilities and catch specific exceptions.
+          LOG_GENERAL(
+              WARNING,
+              "Exception thrown trying to update balance on contract Account "
+                  << e.what());
+        }
+        try {
+          if (it->Nonce().size())
+            targetAccount->SetNonce(std::stoull(it->Nonce()));
+        } catch (std::exception& e) {
+          // for now catch any generic exceptions and report them
+          // will exmine exact possibilities and catch specific exceptions.
+          LOG_GENERAL(
+              WARNING,
+              "Exception thrown trying to set Nonce on contract Account "
+                  << e.what());
+        }
+        m_storageRootUpdateBufferAtomic.emplace(it->Address());
+      }
+    }
+  }
+
+  if (invoke_type == RUNNER_CREATE)
+    contractAccount->SetImmutable(DataConversion::StringToCharArray(
+                                      "EVM" + evmReturnValues.ReturnedBytes()),
+                                  contractAccount->GetInitData());
+
+  if (not csUpdate) {
+    LOG_GENERAL(WARNING, "EvmUpdateContractStateAndAccount did not succeed");
+    ret = false;
+  }
+
+  return gas;
+}
+
 template <class MAP>
 bool AccountStoreSC<MAP>::UpdateAccounts(const uint64_t& blockNum,
                                          const unsigned int& numShards,
                                          const bool& isDS,
                                          const Transaction& transaction,
                                          TransactionReceipt& receipt,
                                          TxnStatus& error_code) {
-  // LOG_MARKER();
-  LOG_GENERAL(INFO, "Process txn: " << transaction.GetTranID());
```

Comment:
This log is a useful one. Might be helpful to keep it

Reply:
Always happy to oblige sir, I will enable it if LOG_SC is on.